src/core/primitives/RadioGroup/RadioGroupPrimitive.stories.tsx

```
@@ -32,9 +32,13 @@
             //  defaultValue='css'
             >
                 {options.map((option) => (
-                    <RadioGroupPrimitive.Item key={option.id} value={option.value}>
+                    <label>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)